event.go

```diff
@@ -21,10 +21,12 @@ var eventPool = &sync.Pool{
 // Event represents a log event. It is instanced by one of the level method of
 // Logger and finalized by the Msg or Msgf method.
 type Event struct {
-	buf   []byte
-	w     LevelWriter
-	level Level
-	done  func(msg string)
+	buf      []byte
+	w        LevelWriter
+	level    Level
+	done     func(msg string)
+	h        []Hook
+	hasLevel bool
```

Comment:
What about adding a new `NoLevel` level before `DebugLevel` in the enum so we don't need this extra field?

Reply:
I thought about doing it, but wanted to change as little as possible. I'll send some more commits your way.

Reply:
It would let us remove the ugly `addLevelField` param too.

Reply:
This part is done. The code is simpler now (benchmarks say it is faster, too).
After the merge could you please handle the NoLog case in console.go?
Please take a look at syslog.go, too.